support-frontend/assets/pages/subscriptions-landing/copy/subscriptionCopy.tsx

```diff
@@ -60,6 +94,67 @@ function getGuardianWeeklyOfferCopy(discountCopy: string) {
 	return '';
 }
 
+function buildDigialPlusBenefits(): ProductBenefit[] {
+	const benefits = [
+		'<strong>The Guardian Editions app</strong> including Guardian newspaper, Guardian Weekly and the Long Read on your mobile and tablet',
+		'Unlimited access to the <strong>Guardian app</strong> and <strong>Guardian Feast app</strong>',
+		'Digital access to the Guardian’s 200 year <strong>newspaper archive</strong>',
+		'<strong>Ad-free reading</strong> on all your devices',
+	];
+	return benefits.map((benefit) => ({ copy: benefit }));
+}
+
+function digitalPlus(
+	countryGroupId: CountryGroupId,
+	priceCopy: PriceCopy,
+): ProductCopy {
+	return {
+		title: 'Enjoy our suite of editions with&nbsp;<mark>Digital Plus</mark>',
+		subtitle: getDigitalPlusPrices(countryGroupId),
+		description: 'Enjoy our suite of editions with Digital Plus',
+		buttons: [
+			{
+				ctaButtonText: getDigitalPlusDisplayPrice(
```

Comment:
think this comes back with `Monthly/Annual` where as the figma is (annoyingly!) `monthly/annually`?